extensions/kafka/src/main/java/com/hazelcast/jet/kafka/impl/HazelcastAvroSerde.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.kafka.impl;
+
+import org.apache.avro.Schema;
+
+import java.util.Map;
+
+public abstract class HazelcastAvroSerde {
```

Comment:
8d2a72b

Reply:
Sorry I should've elaborated, the point is that this abstract class does not carry any data used by its descendants and its only method takes all the data it needs through its args, so it looks more like an utility class rather than abstract class.